QuickDeployment/QuickDeploymentDesktopReadme.md

```diff
@@ -130,19 +146,21 @@ We recommend you use Google Chrome to interact with all Web interfaces for the d
 ### SSL Information
 
 All services have been protected with Self-Signed SSL certificates and are placed in the appropriate stores. Under the following situations you would want to run the script that follows:
-* If you want to expose this to the internet so clients can connect from outside your network
-* If you change the hostname of this server
-* If you add the QDE to a domain
-* If you would like to use your own SSL/TLS certificates
 
+- If you want to expose this to the internet so clients can connect from outside your network
+- If you change the hostname of this server
+- If you add the QDE to a domain
+- If you would like to use your own SSL/TLS certificates
 
 ```powershell
 Set-ExecutionPolicy Bypass -Scope Process -Force ; . C:\choco-setup\files\New-SslCertificates.ps1
```

Comment:
Be consistent on the one line or two.

Reply:
Which do we want? Personally I thoroughly dislike the oneliner approach, but whichever we want I'll standardize it to.

Reply:
I'm a one-liner kind of guy. Makes copy paste easier.

Reply:
Harder to mess up copying one line than it is multiple. Just sayin, I've seen it firsthand. 🤦

Reply:
Yeah, that's definitely fair given the variance of multiline paste support. I don't expect it to be much of an issue tbh, but we can keep it "one line" for that kind of edge case.